### PR TITLE
Add owostat.us to the FAQ

### PR DESCRIPTION
**Pull request by DoNotSpamPls**
@itsaderm ARE YOU HAPPY NOW???

### CODE REVIEW

**Comment by aurieh** [comment]
needs noscript support first

also I don't like the wording - domains can't really be "down"

**Comment by itsaderm** [comment]
i am happy!!!!

**Comment by DoNotSpamPls** [comment]
noscript support will come in another PR

File: src/views/faq.pug [hunk 1]
```diff
@@ -160,6 +160,11 @@ block content
       li #[code *.while-at.school]
       li #[code *.while-at.work]
 
+    p.
+      Some of these domains may not be working when you are reading this (expired or some
+      other issue), so feel free to check the community website #[a(href='https://owostat.us') https://owostat.us] 
+      to make sure that the domain you've chosen is online at the moment.
+
```
Comment (deansheather) [inline src/views/faq.pug]: non trimmed whitespace

File: src/views/faq.pug [hunk 2]
```diff
@@ -160,6 +160,11 @@ block content
       li #[code *.while-at.school]
       li #[code *.while-at.work]
 
+    p.
```
Comment (deansheather) [inline src/views/faq.pug]: I don't like this wording.

```pug
      Some of these domains may be broken when you go to use them (they
      #[em are] donated, after all). A website called
      #[a(href='https://owostat.us' ref='nofollow' target='_blank') owostat.us]
      has been created by the community to mirror the list on this website with
      green or red colors to signal whether the domain is working with owo or
      not.
```
Even this wording isn't that great to be honest.

**Pull request merged**